### PR TITLE
Because of noticeable performance issue made gzip optional, default disabled

### PR DESCRIPTION
**Pull request by dylang**
Changed minmax api to match suggestion proposed by @Cowboy:

gruntjs/grunt-contrib-uglify/pull/35

Will be updating that PR to match this API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,21 +33,31 @@ Convert an object to an array of CLI arguments, which can be used with `child_pr
 
 Strip a path from a path. normalize both paths for best results.
 
-#### minMaxInfo(min, max)
+#### minMaxInfo(min, max, report)
 
 Helper for logging compressed, uncompressed and gzipped sizes of strings.
 
+#### report
+Choices: `false`, `min`, `gzip`
```
Comment (cowboy) [inline README.md]: Do you mean: `false`, `'min'`, `'gzip'`?

File: README.md [hunk 2]
```diff
@@ -33,21 +33,31 @@ Convert an object to an array of CLI arguments, which can be used with `child_pr
...
+Either do not report anything, report only minification result, or report minification and gzip results.
+
+**Important** Including `gzip` results can make this task 5-10x slower depending on the size of the file.
+
+
 ```js
 var max = grunt.file.read('max.js');
 var min = minify(max);
-minMaxInfo(min, max);
+minMaxInfo(min, max, 'min', 'gzip');
```
Comment (cowboy) [inline README.md]: Should this line be `minMaxInfo(min, max, 'gzip');` ?

**Comment by cowboy** [comment]
@dylang I made a few comments inline on your commit.

File: README.md [hunk 3]
```diff
@@ -33,21 +33,31 @@ Convert an object to an array of CLI arguments, which can be used with `child_pr
...
 
 Helper for logging compressed, uncompressed and gzipped sizes of strings.
 
+#### report
+Choices: `false`, `min`, `gzip`
+Default: `false`
+
+Either do not report anything, report only minification result, or report minification and gzip results.
+
+**Important** Including `gzip` results can make this task 5-10x slower depending on the size of the file.
```
Comment (cowboy) [inline README.md]: `gzip` -> `'gzip'`

**Comment by dylang** [comment]
Updated readme to quote `'min'` and `'gzip'`, thanks @cowboy for reviewing and feedback.

**Comment by dylang** [comment]
Anything else I can do to help get these PR's in?

**Comment by tkellen** [comment]
ping @cowboy

**Comment by cowboy** [comment]
The changes seem good to me.

**Pull request closed without merging**